### DISCUSSION TITLE
Identify and collect annotations associated with a tag

### DISCUSSION THREAD

**Discussion by lpurban-f** [windingwind/zotero-better-notes#1184] [Q&A]
Hello - A couple of years ago, in Zotero 6, when I first used BetterNotes, there was a way that I could use a template to locate and collect annotations associated with a particular tag. In the updated version in use with Zotero 7, I cannot figure out how to do that. Has anyone else set this up? I see that templates can be edited, but I'm not very technical (don't write code), so I'm not sure how to go about setting something up for that. 
Thanks in advance for your help.

**Comment by windingwind** [comment]
https://github.com/windingwind/zotero-better-notes/discussions/241

**Reply by lpurban-f** [reply]
Thank you, @windingwind! I was able to copy/import the template, and I can now gather a collection of **all annotations**, sorted by tag. However, I cannot figure out how (or if there is a way) to only gather & extract annotations for just one tag. Is there a way to do that? I tried doing a search and saving that search, but if I then try to use the template with the saved search, I get an error that says "Please select a valid collection."
